### PR TITLE
Add image metadata: io.resin.architecture, io.resin.device-type and io.resin.qemu.version

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -6,6 +6,8 @@ SUITES='wheezy jessie'
 MIRROR='http://archive.raspbian.org/raspbian'
 REPO='resin/rpi-raspbian'
 LATEST='jessie'
+ARCH='rpi'
```
Comment (petrosagg) [inline build.sh]: Why not `armv6hf`?

File: build.sh [hunk 2]
```diff
@@ -6,6 +6,8 @@ SUITES='wheezy jessie'
...
+QEMU_VERSION='2.5.0'
```
Comment (petrosagg) [inline build.sh]: This should be `v2.5.0-resin`

File: build.sh (hunk 1 above)
Comment (nghiant2710) [inline build.sh]: @petrosagg, we use `rpi` as the arch for Raspberry PI 1

File: build.sh (hunk 2 above)
Comment (nghiant2710) [inline build.sh]: Updated!

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**